packages/medusa/src/api/routes/admin/discounts/list-discounts.js

```diff
@@ -34,6 +34,10 @@ export default async (req, res) => {
       selector.is_dynamic = req.query.is_dynamic === "true"
     }
 
+    if ("is_disabled" in req.query) {
+      selector.is_disabled = req.query.is_disabled === "true"
+    }
+
```

Comment:
Nice! Could we prepare for when we need to expand the filtering on discounts by using a more generic approach like: https://github.com/medusajs/medusa/blob/329f8b134c6416bc2ed5e8a8099055223ac05aab/packages/medusa/src/api/routes/admin/orders/list-orders.js#L57

Let me know, what you think :)